onecodex/analyses.py

```diff
@@ -246,6 +254,14 @@ def _metadata_fetch(self, metadata_fields, label=None):
 
         return magic_metadata, magic_fields
 
+    @property
+    def all_nan_classification_ids(self):
+        if self.__class__.__name__ == "OneCodexAccessor":
```

Comment:
Add a comment that this case happens when `self` is a `ClassificationsDataFrame`